python/packages/nisar/antenna/pattern.py

```diff
@@ -400,27 +400,27 @@ def form_pattern(self, tseq, slant_range: Linspace,
             if tx_pol == "L":
                 tx_bmf_pat[tx_pol] = (
                     self.tx_bmf['H'].form_pattern(
-                        t, slant_range, nearest=nearest,
+                        tgroup, slant_range, nearest=nearest,
```

Comment:
I believe the time should be `tseq` not `tgroup` for all the lines here and below. `tgroup` is a portion of `tsec`. 
```suggestion
                        tseq, slant_range, nearest=nearest,
```

Reply:
Agreed. Thanks for catching that!